### PR TITLE
Fix beta doc links, truncated help menu label

### PR DESCRIPTION
**Pull request by tmarkley**
Signed-off-by: Tommy Markley <markleyt@amazon.com>

### Description
This doesn't fix all links because not all pages exist in the beta docs site. This is a stopgap for the 1.0 release. The following links are addressed:
* Main documentation link from the help menu
* DQL link
* Aggregation help links
* Removes the 'kibana' reference from the remaining broken links

The help menu label for the "OpenSearch Dashboards documentation" option was truncated but that was fixed by expanding the width.

help menu

### Issues Resolved
Addresses some of the links that need to be fixed for https://github.com/opensearch-project/OpenSearch-Dashboards/issues/335
 
### Check List
- [x] New functionality includes testing.
  - [x] All tests pass
- [ ] New functionality has been documented.
  - [ ] New functionality has javadoc added
- [x] Commits are signed per the DCO using --signoff

### CODE REVIEW

File: src/core/public/doc_links/doc_links_service.ts [hunk 1]
```diff
@@ -126,16 +127,16 @@ export class DocLinksService {
           introduction: `${OPENSEARCH_WEBSITE_URL}guide/en/kibana/${DOC_LINK_VERSION}/index-patterns.html`,
         },
         addData: `${OPENSEARCH_WEBSITE_URL}guide/en/kibana/${DOC_LINK_VERSION}/connect-to-elasticsearch.html`,
```
Comment (kavilla) [inline src/core/public/doc_links/doc_links_service.ts:129]: do we want to replaced the "kibana" reference or just delete it to remove the reference or should we verify first if it's fine to leave the word kibana in?

Comment (tmarkley) [reply src/core/public/doc_links/doc_links_service.ts:129]: Yeah I can just remove the reference for now.

**Review by ananzh — DISMISSED** [review]

**Review by ananzh — APPROVED** [review]
LGTM

**Review by kavilla — APPROVED** [review]
Awesome LGTM! We will need to replace these will actual values in the future once we have a substitute.

**Pull request merged**